STYLE.md

```diff
@@ -183,6 +183,27 @@ exceptions =
   :> West :> Middle :> Nil
 ```
 
+### Language pragmas
+Place LANGUAGE pragmas right after a module's documentation. Do not align the `#-}`s. `Safe`, `Unsafe`, or in some way "special" language pragmas should follow the normal ones separated by a single blank line. Pragmas should be ordered alphabetically. Example:
```

Comment:
Perhaps qualify what special means here to avoid later *but `-XBreakEverything` isn't special!*

Reply:
Alright; I'll clarify with "use your judgement". I trust our contributors and reviewers O:).

Reply:
As I see it, there's a non-obvious distinction between special and non-special, and while I don't expect many false negatives, I think there might be some false positives (e.g. `UndecidableInstances` could be considered special as it can make compilation non-terminating).

Is special something potentially dangerous? Something with a drastic change to semantics?

(Sorry for playing Devil's advocate on this point, I just think it's worth considering)

Reply:
Mostly safe/unsafe/trustworthy. Just things that don't fit in the normal way you'd think about language extensions or the things you'd like to conditionally include with CPP. I really don't want to codify this too strictly. If someone has a good reason for putting a language extension separately (for example, they might want to explain why a certain extension is necessary), they should just be able to do it.

Reply:
All right, that seems reasonable :+1: